src/UseCases/offices.js

```diff
@@ -1,37 +1,37 @@
-const Office = require('../Models/office');
+const Offices = require('../Models/offices');
```

Comment:
@KristomeisterDev  Cuando hacen el llamado a un modelo por lo regular es un singular.
```suggestion
const Office = require('../Models/offices');
```

Reply:
lo mismo lo modifique, por lo que me comentaste, ahora lo regreso a como estaba.